requirements.txt

```diff
@@ -1,6 +1,7 @@
-httpretty>=0.8,<=0.8.10
-recurly>=2.2.23
+httpretty>=0.9.4
+recurly>=2.9.0,<3.0.0
```

Comment:
This will require regression testing in the @15five/provisioning-payment-squad . We'll need to get a ticket set up for that.